dir.props

```diff
@@ -485,6 +485,11 @@
       </PropertyGroup>
     </Otherwise>
   </Choose>
+  
+  <!-- Default Test platform to deploy the netstandard compiled tests to -->
+  <PropertyGroup>
+    <TestTFM Condition="'$(TestTFM)'==''>netcoreapp1.0</TestTFM>
```

Comment:
I believe this may work as today we are filtering on TargetGroup which should be empty and will include all tests in the runs. please validate this is happening to ensure no one will be broken. and I can do the filtering later to make it work with the Test TFM and default the filtering to netcoreapp1.0 too.

---
In reply to: [65789977](https://github.com/dotnet/corefx/pull/9129#discussion_r65789977) [](ancestors = 65789977,65789923)